tools/list_webhooks.rb

```diff
@@ -1,14 +1,13 @@
-
-
 Maestrano::Connector::Rails::Organization.all.each do |organization|
-  next unless organization.oauth_uid
-  shop_name = organization.oauth_uid
-  p "#{organization.uid}  #{shop_name} #{organization.name}"
-  token = organization.oauth_token
-  ShopifyAPI::Session.temp(shop_name, token) do
-  ShopifyAPI::Webhook.all.each do |webhook|
-    p webhook.address
-  end
+  begin
+    next unless organization.oauth_uid
+    shop_name = organization.oauth_uid
+    p "#{organization.uid}  #{shop_name} #{organization.name}"
+    token = organization.oauth_token
+    ShopifyAPI::Session.temp(shop_name, token) do
+    ShopifyAPI::Webhook.all.each do |webhook|
+      p webhook.address
+    end
   rescue Exception => e
```

Comment:
Not related to this PR, but I have noticed we rescue `Exception` here without re-raising